src/stores/index.js

```diff
@@ -1,8 +1,12 @@
 import currentUser from './currentUser.js'
+import admins from './admins.js'
```

Comment:
Here as well, please apply the pinia convention: useAdminsStore, useAccountsStore

Reply:
https://pinia.vuejs.org/core-concepts/